module/programs/helper.js

```diff
@@ -1081,14 +1093,17 @@ module.exports = class ProgramsHelper {
               status: httpStatusCode.bad_request.status
           }
         }
-        console.log("kafka push status :",pushProgramUsersDetailsToKafka)
+        
         let joinProgramDetails = joinProgram.toObject();
-  
+        // if programUsersDetails[0].consentShared === true which means the data is already pushed to Kafka once, So revert pushProgramUsersDetailsToKafka to false
+        if (programUsersDetails.length > 0 && programUsersDetails[0].hasOwnProperty('consentShared') && programUsersDetails[0].consentShared === true) {
+          pushProgramUsersDetailsToKafka = false;
+        }
+        console.log("kafka push status :",pushProgramUsersDetailsToKafka)
```

Comment:
@VISHNUDAS-tunerlabs let's remove all extra console we added in 6.0 with this PR

Reply:
@aks30 ok